server/views/pages/appointments/partials/appointment-journey-title.njk

```diff
@@ -1,3 +1,7 @@
 {% macro appointmentJourneyTitle(pageName, appointmentJourney) -%}
-  {{ pageName }}
+  {% if appointmentJourney.mode == AppointmentJourneyMode.EDIT %}
+    {{ ("Repeating " if appointmentJourney.repeat == YesNo.YES else "One-off ") + pageName }}
+  {% else %}
```

Comment:
Aha - here's the leading capitalisation - but some titles are providing a further capitalised sentence and some are not?

Reply:
With hindsight, this is confusing. If a page is in edit mode, the title will always be prefixed with either "Repeating " or "One-off ". This means that passing in a lower case page title is correct so we get "One-off change extra information" instead of "One-off Change extra information". The pages that appear to not have a capitalised title are pages that are only used in edit mode.

May have made more sense to have that partial correct whatever casing was passed in...